### DISCUSSION TITLE
Unable to upgrade using the latest compose

### DISCUSSION THREAD

**Discussion by garnser** [netbox-community/netbox-docker#409] [Getting Help] [answered]
## Current Behavior

I've a current installation of netbox wheras the devel branch has been used for deployments. In this scenario I've tested devel and tags/1.0.0.

As docker-compose is initiated the following issue occurs:

```
docker-compose pull netbox
ERROR: The Compose file is invalid because:
Service nginx has neither an image nor a build context specified. At least one must be provided.
```

## Expected Behavior

Expected that docker-compose shouldn't complain about the nginx service given that it's been aggregated as per d2733917732fd8c73239f50407046604f03c838d

## Debug Information

```docker-compose pull netbox
ERROR: The Compose file is invalid because:
Service nginx has neither an image nor a build context specified. At least one must be provided.
```

The output of `docker-compose logs netbox`:

```text
ERROR: The Compose file is invalid because:
Service nginx has neither an image nor a build context specified. At least one must be provided.
```

**Comment by tobiasge** [comment, marked as answer]
You probably have the nginx service in your `docker-compose.override.yml`. There is a description of what must be changed in the release notes.

**Reply by nicklegger** [reply]
Just found out what needed changed after spending 1hr reading and upgrading and missing this step. lol

**Reply by ahdfarouk** [reply]
what was the change needed ?

**Reply by jxavier527** [reply]
> what was the change needed ?

git clone -b release https://github.com/netbox-community/netbox-docker.git
cd netbox-docker
tee docker-compose.override.yml <<EOF
version: '3.4'
services:
  **netbox:**
    ports:
      - 8000:8080
